doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseTabBar.md

```diff
@@ -164,20 +161,186 @@ The Navigation capabilities offered by Uno.Extensions include regions. Regions a
+                    uen:Region.Attached="True"
+                    VerticalAlignment="Bottom">
+            <utu:TabBar.Items>
+                <utu:TabBarItem uen:Region.Name="One" 
+                                Style="{StaticResource MaterialBottomTabBarItemStyle}" />
+                <utu:TabBarItem uen:Region.Name="Two" 
+                                Style="{StaticResource MaterialBottomTabBarItemStyle}" />
+                <utu:TabBarItem uen:Region.Name="Three" 
+                                Style="{StaticResource MaterialBottomTabBarItemStyle}" />
+            </utu:TabBar.Items>
+        </utu:TabBar>
+    </Grid>
+    ```
+
+* Specify the navigator type as `Visibility` using the `Region.Navigator` attached property on the containing element of your collapsed content `Grid` definitions:
+
+    ```xml
+    <Grid uen:Region.Attached="True"
+        uen:Region.Navigator="Visibility"
+        Grid.Row="1">
+    ```
+
+#### Navigating to Page elements
+
+* You may want to navigate to a `Page` view element represented by a route name. It is possible to do this without defining a view element alongside the other content regions. For instance, you may need to display a subscription sign up page `SignUpPage` which will be defined in a separate XAML file.
+
+* Add a new **Page** item to your app called `SignUpPage` with the following code:
+
+  ```xml
+  <Page
+      x:Class="UsingTabBar.Views.SignUpPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:Uno.Extensions.Navigation.UI.Samples"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+  
+      <Grid>
+            <StackPanel Orientation="Vertical">
+                <TextBlock Text="Benefits of subscribing:"
+                            FontSize="24"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center" />
+                <Button Content="Sign Up"
+                        HorizontalAlignment="Center"
+                        VerticalAlignment="Center" />
+            </StackPanel>
+      </Grid>
+  </Page>
+  ```
+
+* For the purposes for this tutorial, `SignUpPage` will be associated with its own view model `SignUpViewModel`. Add a new **Class** item to your app called `SignUpViewModel` with the following code:
+
+  ```csharp 
+  namespace UsingTabBar.ViewModels;
+  
+  public class SignUpViewModel
+  {
+      public SignUpViewMode()
+      {
+
+      }
+  }
+  ```
+
+* Register `ViewMap` and `RouteMap` instances inside the `RegisterRoutes` method in `AppHead.xaml.cs`. This associates the `SignUpPage` described above with `SignUpViewModel`, as well as avoiding the use of reflection for route discovery.
```

Comment:
That should be `App.cs`, I think

Reply:
Yes, in the new template it should be in app.cs